tests/pdctl/hot/hot_test.go

```diff
@@ -100,4 +91,31 @@ func (s *hotTestSuite) TestHot(c *C) {
 	c.Assert(hotStores.BytesReadStats[1], Equals, bytesRead/10)
 	c.Assert(hotStores.KeysWriteStats[1], Equals, keysWritten/10)
 	c.Assert(hotStores.KeysReadStats[1], Equals, keysRead/10)
+
+	// test hot region
+	statistics.Denoising = false
+	checkInMap := assert.New(c)
+	reportInterval := uint64(3) // need to be minHotRegionReportInterval < reportInterval < 3*RegionHeartBeatReportInterval
+	args = []string{"-u", pdAddr, "config", "set", "hot-region-cache-hits-threshold", "0"}
+	_, _, err = pdctl.ExecuteCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+
+	testHot := func(hotRegionID, hotStoreId uint64, hotType string) {
+		args = []string{"-u", pdAddr, "hot", hotType}
+		_, output, e := pdctl.ExecuteCommandC(cmd, args...)
+		hotRegion := statistics.StoreHotRegionInfos{}
+		c.Assert(e, IsNil)
+		c.Assert(json.Unmarshal(output, &hotRegion), IsNil)
+		checkInMap.Contains(hotRegion.AsLeader, hotStoreId)
+		c.Assert(hotRegion.AsLeader[hotStoreId].RegionsCount, Equals, 1)
+		c.Assert(hotRegion.AsLeader[hotStoreId].RegionsStat[0].RegionID, Equals, hotRegionID)
+	}
+
+	hotReadRegionID, hotWriteRegionID, hotStoreId := uint64(3), uint64(2), uint64(1)
+	pdctl.MustPutRegion(c, cluster, hotReadRegionID, hotStoreId, []byte("b"), []byte("c"), core.SetReadBytes(1000000000), core.SetReportInterval(reportInterval))
+	pdctl.MustPutRegion(c, cluster, hotWriteRegionID, hotStoreId, []byte("c"), []byte("d"), core.SetWrittenBytes(1000000000), core.SetReportInterval(reportInterval))
+	time.Sleep(5 * time.Second)
```

Comment:
Can we reduce the time to sleep?

Reply:
I have tried 2 seconds, but failed. It is 3 seconds now.